projects/packages/search/src/instant-search/class-instant-search.php

```diff
@@ -661,6 +661,12 @@ protected function get_preconfig_widget_options() {
 			);
 		}
 
+		$settings['filters'][] = array(
+			'name'  => '',
+			'type'  => 'author',
+			'count' => 5,
+		);
+
```

Comment:
Hi @dognose24, I guess you just need to remove this section to make E2E tests work - the function is used for preconfigured widgets, automatically added when user first purchase search. I guess, author doesn't need to be on the default filter list?

Reply:
And in E2E testing, the search requests are intercepted, and there's no author aggregations in there, which then breaks the frontend (I believe it shouldn't, it's the same case for other filters too, so not particularly for author)

https://github.com/Automattic/jetpack/blob/3f8ec783e2ca64caffa191c9ad7bf4d4ab95d6ba/projects/plugins/search/tests/e2e/helpers/search-helper.js#L52